soh/soh/Enhancements/randomizer/3drando/shops.cpp

```diff
@@ -338,138 +338,138 @@ void InitTrickNames() {
+     Text{"Progressive Bait Bag", "Sac à Appâts (prog.)", "Bolsa de cebo progresiva"},
+     Text{"Progressive Pear Capacity", "Capacité de poire (prog.)", "Capacidad progresiva de pera"}};
   trickNameTable[0x88] = {
      Text{"Progressive Stick Pack", "Paquet de bâtons Mojo (prog.)", "Mayor capacidad de bastones"},
-     Text{"Progressive Stick Bag", "Sac de bâtons (prog.)", "Mayor capacidad de ramas deku"},
+     Text{"Progressive Branch Capacity", "Capacité de la succursale (prog.)", "Capacidad progresiva de la sucursal"},
      Text{"Progressive Rod Capacity", "Capacité de tiges (prog.)", "Mayor capacidad de cetros deku"}};
   trickNameTable[0x89] = {
      Text{"Progressive Bomblings", "Bombinsectes (prog.)", "Bombinsectos progresivos"},
-     Text{"Progressive Missiles", "Missiles (prog.)", "Misiles progresivos"},
-     Text{"Progressive Bombchu Bag", "Sac à Bombchu (prog.)", "Bombachus progresivos"}};
+     Text{"Progressive Sentrobe Bombs", "Bombe de Sphérodrone (prog.)", "Bomba de helicobot progresivo"},
+     Text{"Progressive Bomb-ombs", "Bombe Soldat (prog.)", "Soldado bomba progresivo"}};
   trickNameTable[0x8A] = {
      Text{"Progressive Stamina Meter", "Jauge d'endurance (prog.)", "Medidor de vigor progresivo"},
-     Text{"Progressive Energy Meter", "Jauge d'énergie (prog.)", "Medidor de energía progresivo"},
-     Text{"Progressive Magic Powder", "Poudre magique (prog.)", "Medidor de carga progresivo"}};
+     Text{"Progressive Energy Gauge", "Jauge d'énergie (prog.)", "Medidor de energía progresivo"},
+     Text{"Progressive Stamina Wheel", "", ""}};
   trickNameTable[0x8B] = {
-     Text{"Progressive Memento", "Souvenir (prog.)", "Silbato progresivo"},
+     Text{"Progressive Whistle", "Siffler (prog.)", "Silbido progresivo"},
      Text{"Progressive Flute", "Flûte (prog.)", "Flauta progresiva"},
      Text{"Progressive Recorder", "Harmonica (prog.)", "Armónica progresiva"}};
   trickNameTable[0xD4] = {
-     Text{"Progressive Titan Blade", "Lame des Titans (prog.)", "Hoja del Titán progresiva"},
-     Text{"Progressive Goron Knife", "Lame Goron (prog.)", "Daga Goron progresiva"},
-     Text{"Progressive Giant Sword", "Épée géante (prog.)", "Espada gigante progresiva"}};
+     Text{"Progressive Darknut Sword", "Épée Darknut (prog.)", "Espada Darknut progresiva"},
+     Text{"Progressive Power Sword", "Épée de puissance (prog.)", "Espada de poder progresiva"},
+     Text{"Progressive Big Stabby", "Gros coup de poignard (prog.)", "Gran puñalada progresiva"}};
 
   trickNameTable[0x0F] = {
-     Text{"Magic Bottle", "Flacon magique", "Frasco feérico"},
-     Text{"Glass Bottle", "Flacon de verre", "Botella de cristal"},
-     Text{"Bottle with Water", "Flacon d'eau", "Botella Tingle"}};
+     Text{"Empty Canteen", "Cantine vide", "cantimplora vacía"},
+     Text{"Vial of Winds", "Fiole de vents", "Vial de Vientos"},
+     Text{"Tingle Bottle", "Flacon de Tingle", "Botella de Tingle"}};
```

Comment:
These ones were forcing the player to pay better attention, feels better to have the old ones